pkg/telemetry/events.go

```diff
@@ -246,6 +246,8 @@ func (t *telemetryService) TrackSubscribeRequested(
 	publisher *livekit.ParticipantInfo,
 ) {
 	t.enqueue(func() {
+		prometheus.RecordTrackSubscribeAttempt()
```

Comment:
Should we be recording the track type in the attempt also? Seeing it recorded for `success` below.

Reply:
due to subscription lifecycle changes, we do not have the track's type during the request. 

so I've removed kind from `promTrackSubscribeCounter`, since we are not using that attribute anywhere. but we would still log it for the current stat, so that we can graph what's currently being subscribed to.